pom.xml

```diff
@@ -92,6 +92,7 @@ THE SOFTWARE.
       <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <jacoco.version>0.7.5.201505241946</jacoco.version>
+        <workflow.version>1.12</workflow.version>
```

Comment:
Why is this here? Seems unused. (Of course you would use it if you wanted to add automated tests.)